src/main.js

```diff
@@ -6,9 +6,6 @@ import languageComponent from "@/components/languageComponent";
 import { createAuth0 } from "@auth0/auth0-vue";
 import authConfig from "../authConfig.json";
 const app = createApp(App);
-// you can create a collection for the different pages, fr and en. in this example it's orders and login.
```

Comment:
Was the comment removed because it was no longer needed?

Reply:
Yes. It was to help the other teammates understand in the beggining